microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -6,13 +6,54 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: webview
-ms.date: 04/23/2024
+ms.date: 05/28/2024
 ---
 # Overview of WebView2 features and APIs
 
 Embedding the WebView2 control in your app gives your app access to various methods and properties that are provided through the WebView2 classes or interfaces.  WebView2 has hundreds of APIs that provide a vast set of capabilities, ranging from enhancing your app's native-platform capabilities, to enabling your app to modify browser experiences.  This article provides a high-level grouping of the WebView2 APIs to help you understand the different things you can do using WebView2.
 
 
+<!-- ====================================================================== -->
+## INCOMING
+
+
+<!-- ------------------------------ -->
+#### Support for the Fluent Style Overlay Scrollbar
```

Comment:
Where should section "Support for the Fluent Style Overlay Scrollbar" go?  See outline at top of https://github.com/MicrosoftDocs/edge-developer/pull/3137
@victorhuangwq @peiche-jessica

Reply:
Placed below **Browser features** > **Status bar**
rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3137&tabs=dotnetcsharp#fluent-style-overlay-scrollbar

Reply:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3137&tabs=dotnetcsharp#fluent-overlay-scrollbars